docs/source/models.rst

```diff
@@ -45,13 +45,16 @@ format. For example, :py:mod:`mlflow.sklearn` outputs models as follows:
 
 ::
 
-    # Directory written by mlflow.sklearn.save_model(model, "my_model")
-    my_model/
+    # Run `python examples/sklearn_logistic_regression/train.py` from mlflow directory
+    model/
```

Comment:
I'd prefer the original description. If I was a user, I would wonder "where does examples/sklearn_logistic_regression/train.py come from?".